packages/primitives/src/Box/Box.tsx

```diff
@@ -76,23 +89,20 @@ export function sx(props: any): SerializedStyles {
 export interface VariantProps {
   theme: Theme
   variant?: string
+  variantKey?: string
```

Comment:
👍 nice, can we add to the Box type as well to be able to pass in the key if we need to?

Reply:
Yes, absolutely.

Reply:
cool, if we're using variant as our base block we won't need to export the key just yet so we can wait till we need it.